wremnants/include/muon_efficiencies_binned_vqt_real.h

```diff
@@ -0,0 +1,475 @@
+#ifndef WREMNANTS_MUON_EFFICIENCIES_BINNED_VQT_REAL_H
+#define WREMNANTS_MUON_EFFICIENCIES_BINNED_VQT_REAL_H
+
+#include <boost/histogram/axis.hpp>
+#include "TFile.h"
+#include "TH3D.h"
+#include <vector>
+#include <array>
+#include <string>
+
+TH3D* iso3d;
```

Comment:
Are we sure the usage of global variables to store the histogram is thread safe?
This might probably be a private or protected member of the class, since I don't think it would be used outside of it anyway

Reply:
That's what I did initially, but for some reason I couldn't get it to work. I either got segfaults or unresolved symbol error. We are reading from the histogram, not writing to it, what is it that might not be thread safe?

Reply:
If you're reading it only it should be thread safe, but then better to make it const. Indeed the helpers can be a huge pain to implement, but they are cleaner. I'm ok with using the global pointers here but would prefer to use the helpers in the future.

Reply:
Hello, I am trying to implement this in the helper directly again.